samples/tab-channel-group/mvc-csharp/README.md

```diff
@@ -29,6 +29,17 @@ In this quickstart we'll walk-through creating a custom channel/group tab with A
   
 - [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
 
+## Run the app (Using Teams Toolkit for Visual Studio)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio.
+1. Install Visual Studio 2022 **Version 17.8 Preview 4 or higher** [Visual Studio](https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-preview#1790-pre10--visual-studio-2022-version-179-preview-1)
```

Comment:
There is stable Version 17.8 now. So we could just say **Version 17.8 or higher**. . Also for other README.

Reply:
Fixed